cirq-core/cirq/transformers/gauge_compiling/cz_gauge.py

```diff
@@ -22,24 +24,31 @@
 from cirq import ops
 
+_gauge_parameters: List[Dict[str, ops.Gate]] = [
+    {"pre_q0": ops.I, "pre_q1": ops.I, "post_q0": ops.I, "post_q1": ops.I},
+    {"pre_q0": ops.I, "pre_q1": ops.X, "post_q0": ops.Z, "post_q1": ops.X},
+    {"pre_q0": ops.I, "pre_q1": ops.Y, "post_q0": ops.Z, "post_q1": ops.Y},
+    {"pre_q0": ops.I, "pre_q1": ops.Z, "post_q0": ops.I, "post_q1": ops.Z},
+    {"pre_q0": ops.X, "pre_q1": ops.I, "post_q0": ops.X, "post_q1": ops.Z},
+    {"pre_q0": ops.X, "pre_q1": ops.X, "post_q0": ops.Y, "post_q1": ops.Y},
+    {"pre_q0": ops.X, "pre_q1": ops.Y, "post_q0": ops.Y, "post_q1": ops.X},
+    {"pre_q0": ops.X, "pre_q1": ops.Z, "post_q0": ops.X, "post_q1": ops.I},
+    {"pre_q0": ops.Y, "pre_q1": ops.I, "post_q0": ops.Y, "post_q1": ops.Z},
+    {"pre_q0": ops.Y, "pre_q1": ops.X, "post_q0": ops.X, "post_q1": ops.Y},
+    {"pre_q0": ops.Y, "pre_q1": ops.Y, "post_q0": ops.X, "post_q1": ops.X},
+    {"pre_q0": ops.Y, "pre_q1": ops.Z, "post_q0": ops.Y, "post_q1": ops.I},
+    {"pre_q0": ops.Z, "pre_q1": ops.I, "post_q0": ops.Z, "post_q1": ops.I},
+    {"pre_q0": ops.Z, "pre_q1": ops.X, "post_q0": ops.I, "post_q1": ops.X},
+    {"pre_q0": ops.Z, "pre_q1": ops.Y, "post_q0": ops.I, "post_q1": ops.Y},
+    {"pre_q0": ops.Z, "pre_q1": ops.Z, "post_q0": ops.Z, "post_q1": ops.Z},
+]
+
 CZGaugeSelector = GaugeSelector(
     gauges=[
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.I, pre_q1=ops.I, post_q0=ops.I, post_q1=ops.I),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.I, pre_q1=ops.X, post_q0=ops.Z, post_q1=ops.X),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.I, pre_q1=ops.Y, post_q0=ops.Z, post_q1=ops.Y),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.I, pre_q1=ops.Z, post_q0=ops.I, post_q1=ops.Z),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.X, pre_q1=ops.I, post_q0=ops.X, post_q1=ops.Z),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.X, pre_q1=ops.X, post_q0=ops.Y, post_q1=ops.Y),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.X, pre_q1=ops.Y, post_q0=ops.Y, post_q1=ops.X),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.X, pre_q1=ops.Z, post_q0=ops.X, post_q1=ops.I),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.Y, pre_q1=ops.I, post_q0=ops.Y, post_q1=ops.Z),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.Y, pre_q1=ops.X, post_q0=ops.X, post_q1=ops.Y),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.Y, pre_q1=ops.Y, post_q0=ops.X, post_q1=ops.X),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.Y, pre_q1=ops.Z, post_q0=ops.Y, post_q1=ops.I),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.Z, pre_q1=ops.I, post_q0=ops.Z, post_q1=ops.I),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.Z, pre_q1=ops.X, post_q0=ops.I, post_q1=ops.X),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.Z, pre_q1=ops.Y, post_q0=ops.I, post_q1=ops.Y),
-        ConstantGauge(two_qubit_gate=CZ, pre_q0=ops.Z, pre_q1=ops.Z, post_q0=ops.Z, post_q1=ops.Z),
+        ConstantGauge(
```

Comment:
keep this code as is and just add the extra parameter

Reply:
Done.